MiniApp/Classes/core/RealMiniApp.swift

```diff
@@ -194,6 +194,25 @@ internal class RealMiniApp {
             }
         }
     }
+
+    func retrieveMiniAppMetaData(appId: String,
+                                 version: String? = nil,
```

Comment:
Why is `version` optional here? Below it looks like we just call return `.failure` if version is `nil`, so couldn't we just make it a required param?

Reply:
When I started the ticket I was updating the `MiniApp.info` which needs the `version` to be optional. Now that we have separate interface to get the manifest, I changed it.